storey/dtypes.py

```diff
@@ -64,6 +64,14 @@ def sort_windows_and_convert_to_millis(windows):
 
 
 class FixedWindows(WindowsBase):
+    """
+        List of time windows representing fixed time intervals.
```

Comment:
Indentation of docstring here differs from docstrings added to storey/aggregations.py in this PR.